### ISSUE TITLE
Provider not finding super fresh dashboards

### ISSUE THREAD

**Issue by alrosot** [grafana/terraform-provider-grafana#33]
Hi there,
I found an inconsistent behaviour in the provider that I believe to be a bug.

### Terraform Version
Terraform v0.11.8

### Affected Resource(s)
Please list the resources as a list, for example:
- grafana_dashboard

### Terraform Configuration Files
```hcl
resource "grafana_dashboard" "nodes" {
  config_json = "${file("dashboards/Nodes.json")}"
}

(and another dozen of this)
```

### Debug Output
```
2018-10-08T00:49:02.715Z [DEBUG] plugin.terraform-provider-grafana_v1.2.0_x4: 2018/10/08 00:49:02 [WARN] removing dashboard nodes from state because it no longer exists in grafana
grafana_dashboard.nodes: Creation complete after 3s
```

### Expected Behavior
Grafana dashboard should be created in grafana and also in terraform's state

### Actual Behavior
In some circumstances, the dashboard gets created in Grafana, but not in terraform's state

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Create around 12 different dashboards with a total payload of at least ~800k
2. `terraform plan`
3. `terraform apply`
4. `terraform plan`

At step 4, terraform might refresh just a subset of the created dashboards (as some were not created in terraform state). And when it tries to create the 'missing' ones, it fails due to an id clash (error code 412 - precondition)

### Important Factoids
The pipeline was working fine until it reached a certain size threshold. After this point, sometimes it works, but most of the times it fails. And it skips different number of resources (the worst I've seen was missing 6 dashboards, which is half of them)

My personal theory is that the code is querying grafana faster than it can handle the initial create command. 

### References

Maybe if we just give it more time (or more opportunities) to the check below the issue would go away.

[ReadDashboard function](https://github.com/terraform-providers/terraform-provider-grafana/blob/c926ec06683bd7a1cdeae29d57986e045b21c3b2/grafana/resource_dashboard.go#L59)

**Comment by alrosot** [comment]
It seem we can workaround this by disabling parallelism:
`terraform apply -parallelism=1`
The issue probably would still be there, but as this flag dramatically reduces the stress on the API, it is very unlikely to happen.

**Comment by julienduchesne** [comment]
Duplicate of https://github.com/grafana/terraform-provider-grafana/issues/64. This should be fixed using retries (added in v1.15.0) or with @alrosot 's suggestion of `-parallelism=1`
